src/styles/utiles/_functions.scss

```diff
@@ -0,0 +1,10 @@
+@function catalog-width(
```

Comment:
I like this function, yet have a question. Why do you calculate paddings? 
If you would like to center your catalog, just use `margin: 0 auto`, or center by parent element with flex property